scripts/build/builders/bouffalolab.py

```diff
@@ -19,36 +19,43 @@
 from .gn import GnBuilder
 
 
+def raise_exception(info):
+    logging.fatal('*' * 80)
```

Comment:
This seems very specific to this platform - please remove and keep consistent logic and just raise.

If we need special exception formatting, then we should update at a global level in some other file. We are trying to not diverge much between various implementations.

Reply:
Removed